stylus/components/badge.styl

```diff
@@ -0,0 +1,38 @@
+@require '../settings/palette'
+
+radius = .4375rem
+
+$badge-root
+    position relative
+    display inline-flex
+
+$badge
+    position absolute
+    bottom - radius
+    right - radius
+    width radius * 2
+    height radius * 2
+    border-radius 50%
+    background-color black
+    color white
+    z-index 1
+    display flex
+    flex-direction row
+    flex-wrap wrap
+    justify-content center
+    align-content center
+    align-items center
+    font-size .625rem
+    border 1px solid white
```

Comment:
Even for border, I prefer using `rem` so  it should be `.0625rem` even if, I know, it isn't read easily. :)

Reply:
Yeah I missed this one